src/components/Layout/Header/GlobalSearch/ActionResults/ActionResults.tsx

```diff
@@ -0,0 +1,41 @@
+import { List } from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+
+import { ListItemSection } from '../ListItemSection'
+import type { GlobalSearchResultsProps } from '../types'
+import { ActionResult } from './ActionResult'
+
+export const ActionResults: React.FC<GlobalSearchResultsProps> = ({
```

Comment:
We could define this input type more strictly since we know what we are passing to avoid the casting on results type

Reply:
ddc5282a2